docs/projects/template-fps/overview.md

```diff
@@ -6,37 +6,6 @@ The FPS starter project is a great starting point for working on the SpatialOS G
 
 The SpatialOS GDK can help you run your games at scale, making it easy for even a small team to make a massive multiplayer hit game. The FPS starter project is already configured to scale-up, and you can test it out right now! Check out the [FPS at scale]({{urlRoot}}/welcome) guide to dive right in!
 
```

Comment:
This wants to link to get-started now, I think?

Reply:
good spot